lib/src/cln_grpc_base.dart

```diff
@@ -64,7 +65,8 @@ class GRPCClient {
     channel = ClientChannel(host,
         port: port, options: ChannelOptions(credentials: cred));
     stub = NodeClient(channel);
-    print("Client Successfully Connected to Lightning Server");
+    LogManager.getInstance
+        .debug("Client Successfully Connected to Lightning Server");
```

Comment:
This is misleading!! 
We need to remove or add some "if condition to check" because we are actually not checking if the client is connected or has thrown some error!

Reply:
I remove it!